traffic_ops/traffic_ops_golang/README.md

```diff
@@ -79,6 +79,14 @@ Go is a compiled language so any local changes will require you to CTRL-C the co
 
 `go build && ./traffic_ops_golang -cfg $HOME/cdn.conf -dbcfg ../app/conf/development/database.conf`
 
+To cross compile the code in windows to generate binaryt to run on linux, Please replace GOARCH for architecture to build
```

Comment:
spelling; "binaryt" -> "binary"

Windows and Linux are proper nouns; need capitalization

Grammar: "Please" should not be capitalized in the middle of the sentence

Grammar and clarity: "..., Please replace GOARCH for architecture to build" -> "..., please run the following command, replacing `GOARCH` with the architecture for which to build." (note ending punctuation).